doc/configure

```diff
@@ -57,8 +55,8 @@ RULES = [{
                f" { TEMPLATE_DIR / 'voluptuous-man.jinja.scdoc' }"
                " | scdoc > ${out}"
 }, {
-    "name": "roff_to_html",
-    "description": "converting ${man-name}.roff HTML",
+    "name": "man_to_html",
+    "description": "converting ${man-name}.1 HTML",
```

Comment:
This isn't always going to be `.1`. I suggest to leave the extension out completely, just use `${man-name}`. This is just debugging output so it doesn't really matter

Reply:
Resolved